api/src/main/java/io/strimzi/api/kafka/model/listener/arraylistener/GenericKafkaListenerConfiguration.java

```diff
@@ -229,6 +230,17 @@ public void setIpFamilies(List<IpFamily> ipFamilies) {
         this.ipFamilies = ipFamilies;
     }
 
+    @Description("Whether to create the bootstrap service or not." +
+            "This field can be used with the `loadBalancer` type listener.")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Can you set this to NON_DEFAULT? That way it should not always render into the YAML unless it would be set to false (and thus not change the YAMLs which already exists without the value.)